libnavigator/proguard-rules.pro

```diff
@@ -1 +1,7 @@
 # --- Navigator ---
+
+# Refs https://github.com/mapbox/mapbox-bindgen/issues/27
+-keep class com.google.gson.** {*;}
+-keep class com.mapbox.geojson.** {*;}
+-keep class com.mapbox.bindgen.** {*;}
+-keep class com.mapbox.common.** {*;}
```

Comment:
seems that rule must be a part of https://github.com/mapbox/mapbox-base-android/blob/master/common/build.gradle.kts. Could you make an issue?

Reply:
This issue targets our native common library, not https://github.com/mapbox/mapbox-base-android.

Reply:
it's out of this PR, but the same packages of different libs/sdks leads to confusing 😕 
Anyway, would be great to make `-keep class com.mapbox.common.** {*;}` a part of upstair library

Reply:
For sure, already ticketed that :+1: